client/components/Content/index.js

```diff
@@ -0,0 +1,2 @@
+import Content from "./Content";
+export default Content;
```

Comment:
But you still need that extra `index.js`. Flat is easier to look up as it's grouped by the names already and the js and css files stand next to each other always and with the benefit that you can use `import x from "./x"` instead of `import x from "./x/x"` which I understand is the purpose of this `index.js`. How is a separate directory easier to understand/view the components?